bika/lims/interfaces/__init__.py

```diff
@@ -804,3 +804,18 @@ class INumberGenerator(Interface):
 class IGenerateID(Interface):
     """Marker Interface to generate an ID
     """
+
+class ITopRightListingHook(Interface):
+    """
+    Marker interface to hook list exporters in bikalisting
+    """
+
+class ITopLeftListingHook(Interface):
+    """
+    Marker interface to hook list exporters in bikalisting
+    """
+
+class ITopWideListingHook(Interface):
+    """
+    Marker interface to hook list exporters in bikalisting
```

Comment:
Instead of `to hook list exporters` better `to hook html components in bikalisting`